src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -116,6 +116,13 @@ export default class RightPanel extends React.PureComponent<Props> {
                         disabled={item.isPublished || !isOwner}
                         onChange={name => this.handleChange({ ...item, name })}
                       />
+                      <Input
+                        itemId={item.id}
+                        label={t('global.description')}
+                        value={item.description}
+                        disabled={item.isPublished || !isOwner}
```

Comment:
What do you think of moving `item.isPublished || !isOwner` to a variable like `canEdit`? Seems to be used also in other components.

Reply:
mm I can't create a `canEdit` variable at the root of the `render` method because it depends on `item` that is taken from a lambda within the return value of `render`. I could create a variable within that scope but it's not going to be at the root level with the rest of the variables which i don't think it makes it much clearer tbh